test/uniform/app_test.exs

```diff
@@ -83,19 +83,19 @@ defmodule Uniform.AppTest do
     assert %MixDep{mix_deps: [:indirect_mix]} = app.internal.deps.mix.included_mix
     assert %MixDep{mix_deps: []} = app.internal.deps.mix.indirect_mix
 
-    assert app.internal.deps.included.lib == [
+    assert Enum.sort(app.internal.deps.included.lib) == [
              :always_included_lib,
              :included_lib,
              :indirectly_included_lib
            ]
```

Comment:
So these were the test failures that broke on OTP 26?

Do you know what underlying behavior changed to cause these behaviors? Just curious.

Reply:
> When migrating to Erlang/OTP 26, keep it mind it changes how maps are stored internally and they will be printed and traversed in a different order... [Elixir v1.14.4 release notes](https://github.com/elixir-lang/elixir/releases/tag/v1.14.4)

Remember how maps used to have a stable sort order if the number of items was `n` (32, right) or less?. OTP 26 changes this, so if you have a map that gets converted into a list at some point, the resulting order of those items is non-deterministic.